ArduCopter/takeoff.cpp

```diff
@@ -93,14 +93,13 @@ void Mode::_TakeOff::do_pilot_takeoff(float& pilot_climb_rate_cm)
     }
 }
 
-void Mode::auto_takeoff_run()
+bool Mode::auto_takeoff_run()
```

Comment:
we should add a comment here explaining when it return true or false.  It looks like it returns true once it has completed the takeoff.